modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -776,25 +776,25 @@
                       "trackedItems": [
                         {
                           "closedDate": null,
-                          "description": "MG-not a recognized condition",
+                          "description": null,
```

Comment:
be nice to have one of these not display `null`, but seems like null is the more commonly seen value. May rework claim.json some day to include a value here.